test/README.md

```diff
@@ -117,6 +117,8 @@ This assertion method requires at least a CSS selector and an image title.
 Pixel tests are generated in five layouts by default: desktop, medium, mobile,
 dark and rtl.
 
+FIXME: #detail-content no longer exists
```

Comment:
That should go :grin:

Reply:
Yeah, the example doesn't need to actually work.